bool t_ready
string task_id
bool t_status
float32 task_x
float32 task_y
float32 task_theta
string rob_id
bool r_status
